### PR TITLE
NXP-27784: Don't export auto properties

### PR DESCRIPTION
**Pull request by covolution**

### CODE REVIEW

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/DataSetBulkAction.java [hunk 1]
```diff
@@ -129,26 +130,31 @@ protected void compute(CoreSession coreSession, List<String> ids, Map<String, Se
...
-                        Record record = toRecord(command.getId(), subDoc);
-                        if (isTraining) {
-                            training.add(record);
-                        } else {
-                            validation.add(record);
-                        }
+            for (DocumentModel doc : loadDocuments(coreSession, ids)) {
+                BlobTextFromDocument subDoc = null;
+                List<String> targetProperties = new ArrayList<>();
+                targetProperties.addAll(customProperties);
```
Comment (andreinechaev) [inline nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/DataSetBulkAction.java]: `ArrayList` constructor accepts another collection to create a copy from it.

Comment (covolution) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/DataSetBulkAction.java]: Yes, I started with that, but had a problem.  One of them was unmodifiable so couldn't copy - it was strange.

Comment (andreinechaev) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/DataSetBulkAction.java]: hm, ok. my bad

Comment (covolution) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/DataSetBulkAction.java]: actually, i didn't understand it but changed the code to be sure.

**Pull request merged**